packages/auth/plugin/src/ios/urlTypes.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  ConfigPlugin,
+  IOSConfig,
+  withInfoPlist,
+  ExportedConfigWithProps,
+} from '@expo/config-plugins';
+import fs from 'fs';
+import path from 'path';
+import plist from 'plist';
+
+// does this for you: https://firebase.google.com/docs/auth/ios/phone-auth#enable-phone-number-sign-in-for-your-firebase-project
+export const withIosCaptchaUrlTypes: ConfigPlugin = config => {
+  return withInfoPlist(config, config => {
```

Comment:
In theory, we don't have to use `withInfoPlist`, because it is an [asynchronous mod]() and since here we don't have any async code, maybe we can keep it simple: 

https://github.com/expo/expo-cli/blob/cdb715365bab7f735200c32971437de187a9953d/packages/config-plugins/src/plugins/ios-plugins.ts#L79